restore caffe2 strides

li-roy opened the pull request:
Summary: The workflow passes after D10150834, so we can restore strides.

Differential Revision: D10220313

ezyang reviewed (approved):
That beautiful moment when you realize that it's easy :)

P.S., a note for the future: the threading fix should be put into a separate patch, to ease bisecters from the future (if the ordering there was actually important). For now, mentioning it in the commit message should be good enough.

li-roy commented:
@pytorchbot retest this please

Commit:
restore caffe2 strides (#12381)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/12381

The workflow passes after D10150834, so we can restore strides.

Reviewed By: ezyang

Differential Revision: D10220313

fbshipit-source-id: 9b943b82a5fec1717af60f4b17fa52eef0be36bd
